ansible/roles/olm_setup/templates/osb-operators.catalogsource.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Not the most important. But since these are no longer templates, I think they should be moved under `olm_setup/files`

Reply:
Since the existing tasks file was using the `template` module, I was keeping the .yaml files in there (it automatically looks in the template dir, don't think it looks in the files dir but I could be wrong).

But anyways, cleaned it up a bit more by taking that out.